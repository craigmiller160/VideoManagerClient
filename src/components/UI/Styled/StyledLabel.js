import styled from 'styled-components';

const StyledLabel = styled.label`
    font-weight: bold;
    margin: 0;
`;
StyledLabel.displayName = 'StyledLabel';

export default StyledLabel;