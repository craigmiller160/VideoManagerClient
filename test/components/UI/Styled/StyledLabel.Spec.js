import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components'
import StyledLabel from 'components/UI/Styled/StyledLabel';

describe('StyledLabel', () => {
    it('renders correctly', () => {
        const component = mount(<StyledLabel/>);
        expect(component).toMatchSnapshot();
    });
});