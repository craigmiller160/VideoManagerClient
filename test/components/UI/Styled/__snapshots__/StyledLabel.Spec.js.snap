// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledLabel renders correctly 1`] = `
<StyledLabel>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "lastClassName": "clbXFz",
          "rules": Array [
            "
    font-weight: bold;
    margin: 0;
",
          ],
        },
        "displayName": "StyledLabel",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "label",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <label
      className="sc-bdVaJa clbXFz"
    />
  </StyledComponent>
</StyledLabel>
`;
